db/rdbms/src/main/resources/mapper/ProcessDefinitionMapper.xml

```diff
@@ -25,7 +25,8 @@
     NAME,
     TENANT_ID,
     VERSION_TAG,
-    VERSION
+    VERSION,
+    NULL as FORM_ID
```

Comment:
When quick-fixing things in RDBMS like this, can you create a follow-up issue with label `component/rdbms` and assign it to Patrick or me? Otherwise we will almost never find out that there is a new field we don't support :)

Reply:
Yes, if it is committed with this PR then I will create a follow-up so it's not forgotten.
I might drop that commit. I just wanted to see a green RDBMS job but it looks like it has other problems.
It doesn't really belong to this PR.